src/DIRAC/Core/Tornado/Server/TornadoService.py

```diff
@@ -121,6 +123,8 @@ def export_streamToClient(self, myDataToSend, token):
     # We also need to add specific attributes for each service
     _monitor = None
 
+    SVC_SECLOG_CLIENT = SecurityLogClient()
```

Comment:
If I understand correctly, it works similarly to `MonitoringClient`, so `SecurityLogClient` can be placed in the same way: initialisation in the `_initMonitoring`.